user/reference/bionic.md

```diff
@@ -104,11 +104,20 @@ The following versions of Docker, version control software and compilers are pre
 
 ## Python support
 
-* Supported Python versions: `2.7`, `3.6` or higher.
-* Pre-installed Python versions: `2.7.15`, `3.6.7`, and `3.7.1`.
-* Python `2.7.15` will be used when no language version is explicitly set.
+Images for all languages have Python 2 and 3 installations provided by the distribution. The Python 2 one has pip preinstalled.
```

Comment:
Given that Python 2 sunsets in 96 days, how & when will the upgrade to pip3 be done?  It is not helpful to the community that Travis continues to inflate the number of legacy Python installs in tools like https://pypistats.org

Reply:
> when will the upgrade to pip3 be done

That's up to the maintainers of Ubuntu which Travis Linux environments are based on.

Reply:
BionicBeaver ReleaseNotes say [Python 2 is no longer installed by default. Python 3 has been updated to 3.6.](https://wiki.ubuntu.com/BionicBeaver/ReleaseNotes#Other_base_system_changes_since_16.04_LTS_)

Reply:
Let's continue this discussion at https://travis-ci.community/. I'm not quite sure what it is that you want to see and this is not related to this PR.